### PR TITLE
Updated percentage premium in Keepers

### LINKED ISSUE #850: [Enhancement]: Keepers percentage premium
### What would you like us to improve?

Make it clear to Keepers users that the premium payment is a percentage and not an absolute amount

### How important it is?

Important

### PR DESCRIPTION
**Pull request by wentzeld**
Updated percentage premium (added %) to make it clear that this is a percentage that is added as requested by Sales

**IMPORTANT: Please do not create a Pull Request without creating an issue first.Any change needs to be discussed before proceeding.**

## Closing issues

closes #850 

...

## Description

Made it clear that this adds a percentage to the cost and not some absolute amount

...

## Changes

- High level
- changes that
- you made
- ...

### CODE REVIEW

File: docs/chainlink-keepers/keeper-economics.md [hunk 1]
```diff
@@ -11,19 +11,19 @@ whatsnext:
 
 ## How Funding Works
 
-Your balance is reduced each time a Keeper executes your `performUpkeep` method. There is no cost for `checkUpkeep` calls. If you want to automate adding funds, you can directly call the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
+Your Upkeep has a LINK (ERC-677) balance and every time a Keeper executes your `performUpkeep` function, your LINK balance will be reduced. You can add funds using the Keepers app or by directly calling the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
```
Comment (ritika-b) [inline docs/chainlink-keepers/keeper-economics.md:14]:
```suggestion
Your Upkeep has a LINK (ERC-677) balance. Every time a Keeper executes your `performUpkeep` function, your LINK balance will be reduced. You can add funds using the Keepers app or by directly calling the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
```

File: docs/chainlink-keepers/keeper-economics.md [hunk 2]
```diff
@@ -11,19 +11,19 @@ whatsnext:
...
+When a Keeper executes your `performUpkeep` function, the Keeper Registry will deduct the Upkeep's total gas cost in LINK plus a percentage premium from your Upkeep’s LINK balance and allocate it to the Keeper’s address. The total gas cost in LINK is the gas price of the transaction multiplied by the sum of the gas used for the transaction and an 80K gas overhead for the Keeper call gas used, which is converted to LINK using Chainlink Data Feeds. The percentage premium is the reward to the Keeper for monitoring and performing your Upkeep. The percentage premium varies by network and is listed in our [Supported Networks](../supported-networks/#configurations) page.
```
Comment (ritika-b) [inline docs/chainlink-keepers/keeper-economics.md:18]:
```suggestion
When a Keeper executes your `performUpkeep` function, the Keeper Registry will deduct the Upkeep's total gas cost in LINK as well as a percentage premium from your Upkeep’s LINK balance and allocate it to the Keeper’s address. The total gas cost in LINK is the gas price of the transaction multiplied by the sum of the gas used for the transaction and an 80K gas overhead for the Keeper call gas used. This is converted to LINK using Chainlink Data Feeds. The percentage premium is to compensate the Keeper for monitoring and performing your Upkeep. The percentage premium varies by network and is listed in our [Supported Networks](../supported-networks/#configurations) page.
```

File: docs/chainlink-keepers/supported-networks.html [hunk 3]
```diff
@@ -42,7 +42,7 @@
 
 # Parameters
 
-- **Payment Premium** (`paymentPremiumPPB`): This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+- **Payment Premium %** (`paymentPremiumPPB`): This percentage premium is the reward to the Keepers Network for monitoring and performing your Upkeep. Every time a transaction is submitted for your Upkeep, your LINK balance is reduced by the cost plus this percentage premium.
```
Comment (ritika-b) [inline docs/chainlink-keepers/supported-networks.html:45]:
```suggestion
- **Payment Premium %** (`paymentPremiumPPB`): This percentage premium compensates the Keepers Network for monitoring and performing your Upkeep. Every time a transaction is submitted for your Upkeep, your LINK balance is reduced by the cost plus this percentage premium.
```

**Review by ritika-b — CHANGES_REQUESTED** [review]

File: docs/chainlink-keepers/keeper-economics.md (hunk 1 above)
Comment (dwightjl) [reply docs/chainlink-keepers/keeper-economics.md:14]: +1

File: docs/chainlink-keepers/keeper-economics.md (hunk 2 above)
Comment (dwightjl) [reply docs/chainlink-keepers/keeper-economics.md:18]: +1

File: docs/chainlink-keepers/keeper-economics.md [hunk 4]
```diff
@@ -11,19 +11,19 @@ whatsnext:
...
 
 ## No node competition
 
 Chainlink Keepers do not compete with one another, but rather work together to ensure all registered Upkeeps are performed. This makes costs more predictable upfront, enabling you to estimate costs based on the expected gas consumption.
 
 ## Minimum balance
 
-The network is designed to perform your work even under conditions where gas prices spike. In order to ensure appropriate payment to the Keepers, your LINK balance must exceed the maximum amount that could be paid for performance.
+The Keepers network is designed to perform your Upkeep even when gas prices spike and the minimum balance in LINK reflects the best estimate of the cost to perform your Upkeep when gas prices spike. To ensure your upkeep is monitored and performed, please ensure your Upkeep's balance is above this minimum balance.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:26]:
```suggestion
The Keepers network is designed to perform your Upkeep even when gas prices spike. The minimum balance in LINK reflects the best estimate of the cost to perform your Upkeep when gas prices spike. To ensure your upkeep is monitored and performed, ensure that your Upkeep's balance is above this minimum balance.
```

File: docs/chainlink-keepers/supported-networks.html (hunk 3 above)
Comment (dwightjl) [reply docs/chainlink-keepers/supported-networks.html:45]: +1

File: docs/chainlink-keepers/keeper-economics.md (hunk 1 above)
Comment (wentzeld) [reply docs/chainlink-keepers/keeper-economics.md:14]: Looks good

**Review by ritika-b — APPROVED** [review]

**Review by dwightjl — COMMENTED** [review]
Chainlink Keepers should be capitalized as a proper noun, but individual keepers and upkeeps are not proper nouns and should be lower case.

File: docs/chainlink-keepers/keeper-economics.md [hunk 5]
```diff
@@ -11,19 +11,19 @@ whatsnext:
 
 ## How Funding Works
 
-Your balance is reduced each time a Keeper executes your `performUpkeep` method. There is no cost for `checkUpkeep` calls. If you want to automate adding funds, you can directly call the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
+Your Upkeep has a LINK (ERC-677) balance. Every time a Keeper executes your `performUpkeep` function, your LINK balance will be reduced. You can add funds using the Keepers app or by directly calling the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:14]:
```suggestion
Your Upkeep has a LINK (ERC-677) balance. Every time a keeper executes your `performUpkeep` function, your LINK balance will be reduced. You can add funds using the Keepers App or by directly calling the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
```

File: docs/chainlink-keepers/keeper-economics.md [hunk 6]
```diff
@@ -11,19 +11,19 @@ whatsnext:
...
+When a keeper executes your `performUpkeep` function, the Keeper Registry will deduct the upkeep's total gas cost in LINK as well as a percentage premium from your Upkeep’s LINK balance and allocate it to the keeper’s address. The total gas cost in LINK is the gas price of the transaction multiplied by the sum of the gas used for the transaction and an 80K gas overhead for the keeper call gas used. This is converted to LINK using Chainlink Data Feeds. The percentage premium is to compensate the keeper for monitoring and performing your Upkeep. The percentage premium varies by network and is listed in our [Supported Networks](../supported-networks/#configurations) page.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:18]:
```suggestion
When a keeper executes your `performUpkeep` function, the Keeper Registry will deduct the upkeep's total gas cost in LINK as well as a percentage premium from your upkeep’s LINK balance and allocate it to the keeper’s address. The total gas cost in LINK is the gas price of the transaction multiplied by the sum of the gas used for the transaction and an 80K gas overhead for the keeper call gas used. This is converted to LINK using Chainlink Data Feeds. The percentage premium is to compensate the keeper for monitoring and performing your Upkeep. The percentage premium varies by network and is listed in our [Supported Networks](../supported-networks/#configurations) page.
```

File: docs/chainlink-keepers/keeper-economics.md [hunk 7]
```diff
@@ -11,19 +11,19 @@ whatsnext:
...
 
 ## No node competition
 
 Chainlink Keepers do not compete with one another, but rather work together to ensure all registered Upkeeps are performed. This makes costs more predictable upfront, enabling you to estimate costs based on the expected gas consumption.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:22]:
```suggestion
Individual keepers do not compete with one another, but rather work together to ensure all registered upkeeps are performed. This makes costs more predictable upfront, enabling you to estimate costs based on the expected gas consumption.
```

File: docs/chainlink-keepers/keeper-economics.md [hunk 8]
```diff
@@ -11,19 +11,19 @@ whatsnext:
...
 
 ## Minimum balance
 
-The network is designed to perform your work even under conditions where gas prices spike. In order to ensure appropriate payment to the Keepers, your LINK balance must exceed the maximum amount that could be paid for performance.
+The Keepers network is designed to perform your upkeep even when gas prices spike. The minimum balance in LINK reflects the best estimate of the cost to perform your upkeep when gas prices spike. To ensure your upkeep is monitored and performed, ensure that your upkeep's balance is above this minimum balance.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:26]:
```suggestion
The Keepers Network is designed to perform your upkeep even when gas prices spike. The minimum balance in LINK reflects the best estimate of the cost to perform your upkeep when gas prices spike. To ensure your upkeep is monitored and performed, ensure that your upkeep's balance is above this minimum balance.
```

File: docs/chainlink-keepers/keeper-economics.md [hunk 9]
```diff
@@ -11,19 +11,19 @@ whatsnext:
...
 
 The minimum balance is calculated using the current fast gas price, the Gas Limit you entered for your Upkeep, the max gas multiplier, and the for conversion to LINK. To find the latest value for the `gasCeilingMultiplier`, see the [Registry Configuration](../supported-networks/#configurations).
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:28]:
```suggestion
The minimum balance is calculated using the current fast gas price, the gas limit you entered for your upkeep, the max gas multiplier, and the for conversion to LINK. To find the latest value for the `gasCeilingMultiplier`, see the [Registry Configuration](../supported-networks/#configurations) page.
```

File: docs/chainlink-keepers/supported-networks.html [hunk 10]
```diff
@@ -42,7 +42,7 @@
 
 # Parameters
 
-- **Payment Premium** (`paymentPremiumPPB`): This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+- **Payment Premium %** (`paymentPremiumPPB`): This percentage premium compensates the Keepers Network for monitoring and performing your upkeep. Every time a transaction is submitted for your upkeep, your LINK balance is reduced by the cost plus this percentage premium.
 - **Block Count Per Turn** (`blockCountPerTurn`): The number of blocks in each turn before the Upkeep is rotated to the next Keeper
 - **Check Gas Limit** (`checkGasLimit`): The maximum amount of gas that can be used by your checkUpkeep for off-chain computation.
 - **Call Gas Limit** (`callGasLimit`): The maximum amount of gas that can be used by the client contract’s performUpkeep for the on-chain transaction. You can set an upper limit on your Upkeep during registration, but this number must not exceed the call gas limit on the KeeperRegistry.
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:48]:
```suggestion
- **Call Gas Limit** (`callGasLimit`): The maximum amount of gas that can be used by the client contract’s `performUpkeep` function for the on-chain transaction. You can set an upper limit on your upkeep during registration, but this number must not exceed the call gas limit on the `KeeperRegistry` contract.
```

**Review by dwightjl — APPROVED** [review]

File: docs/chainlink-keepers/keeper-economics.md [hunk 11]
```diff
@@ -11,21 +11,21 @@ whatsnext:
 
 ## How Funding Works
 
-Your balance is reduced each time a Keeper executes your `performUpkeep` method. There is no cost for `checkUpkeep` calls. If you want to automate adding funds, you can directly call the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
+Your Upkeep has a LINK (ERC-677) balance. Every time a keeper executes your `performUpkeep` function, your LINK balance will be reduced. You can add funds using the Keepers App or by directly calling the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:14]:
```suggestion
Your upkeep has a LINK (ERC-677) balance. Every time a keeper executes your `performUpkeep` function, your LINK balance will be reduced. You can add funds using the Keepers App or by directly calling the `addFunds()` function on the `KeeperRegistry` contract. Anyone can call the `addFunds()` function.
```

File: docs/chainlink-keepers/keeper-economics.md [hunk 12]
```diff
@@ -11,21 +11,21 @@ whatsnext:
...
+When a keeper executes your `performUpkeep` function, the Keeper Registry will deduct the upkeep's total gas cost in LINK as well as a percentage premium from your upkeep’s LINK balance and allocate it to the keeper’s address. The total gas cost in LINK is the gas price of the transaction multiplied by the sum of the gas used for the transaction and an 80K gas overhead for the keeper call gas used. This is converted to LINK using Chainlink Data Feeds. The percentage premium is to compensate the keeper for monitoring and performing your Upkeep. The percentage premium varies by network and is listed in our [Supported Networks](../supported-networks/#configurations) page.
```
Comment (dwightjl) [inline docs/chainlink-keepers/keeper-economics.md:18]:
```suggestion
When a keeper executes your `performUpkeep` function, the Keeper Registry will deduct the upkeep's total gas cost in LINK as well as a percentage premium from your upkeep’s LINK balance and allocate it to the keeper’s address. The total gas cost in LINK is the gas price of the transaction multiplied by the sum of the gas used for the transaction and an 80K gas overhead for the keeper call gas used. This is converted to LINK using Chainlink Data Feeds. The percentage premium is to compensate the keeper for monitoring and performing your upkeep. The percentage premium varies by network and is listed in our [Supported Networks](../supported-networks/#configurations) page.
```

**Review by dwightjl — APPROVED** [review]

**Pull request merged**